src/workerd/api/node/buffer.c++

```diff
@@ -120,10 +119,7 @@ uint32_t writeInto(jsg::Lock& js,
     }
     case Encoding::HEX: {
       KJ_STACK_ARRAY(kj::byte, buf, string.length(js), 1024, 536870888);
-      static constexpr jsg::JsString::WriteOptions options =
-          static_cast<jsg::JsString::WriteOptions>(
-              jsg::JsString::NO_NULL_TERMINATION | jsg::JsString::REPLACE_INVALID_UTF8);
-      string.writeInto(js, buf, options);
+      string.writeInto(js, buf, flags);
```

Comment:
Previously we had REPLACE_INVALID_UTF8 in this path, but now we don't. Is this intentional?